promql/parser/parse.go

```diff
@@ -481,13 +481,15 @@ func (p *parser) mergeMaps(left, right *map[string]interface{}) (ret *map[string
 
 func (p *parser) histogramsIncreaseSeries(base, inc *histogram.FloatHistogram, times uint64) ([]SequenceValue, error) {
 	return p.histogramsSeries(base, inc, times, func(a, b *histogram.FloatHistogram) *histogram.FloatHistogram {
-		return a.Add(b)
+		res, _ := a.Add(b)
+		return res
```

Comment:
To be on the safe side let's keep the original and update the function that takes this argument and check the result, see in `func (p *parser) histogramsSeries(base, inc *histogram.FloatHistogram, times uint64,
	combine func(*histogram.FloatHistogram, *histogram.FloatHistogram) *histogram.FloatHistogram,
) ([]SequenceValue, error) {` later
```suggestion
		return a.Add(b)
```

Reply:
Trying to do this makes test fail, investigating.